ui/app/components/auth/form/base.hbs

```diff
@@ -24,7 +24,7 @@
       @isFullWidth={{true}}
       type="submit"
       class="has-top-margin-xl has-bottom-margin-xl"
-      data-test-auth-submit
+      data-test-save
```

Comment:
IMO `data-test-submit` makes way more sense for forms if you want to generalize form submit buttons - forms do not equal "saving" - especially in Vault. Lots of generating credentials, etc that doesn't yield a created resource

Reply:
Agreed, and amended. Note the change nearly doubled the file size of the PR.